api/v3proposalopts.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2023 Attestant Limited.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import "github.com/attestantio/go-eth2-client/spec/phase0"
+
+// V3ProposalOpts are the options for obtaining v3 proposals.
+type V3ProposalOpts struct {
```

Comment:
This seems missing `builder_boost_factor` param.

Reply:
thanks, it was actually added yesterday to the spec https://github.com/ethereum/beacon-APIs/pull/431, after I did the PR. will do the changes soon.